klaytn/src/adapter.ts

```diff
@@ -0,0 +1,131 @@
+import { Requester, Validator, AdapterError } from '@chainlink/external-adapter'
+import { Config, ExecuteWithConfig, ExecuteFactory, AdapterResponse } from '@chainlink/types'
+import { makeConfig } from './config'
+import { LegacyTransaction } from './types'
+import fs from 'fs'
+
+const Caver = require('caver-js')
+const caver = new Caver(process.env.URL)
+
+const privateKey = process.env.PRIVATE_KEY
```

Comment:
`klaytn/src/config.ts`'s `getDefaultCOnfig` will look for `API_KEY` if you want to use that environment variable name instead.